vyper/evm/opcodes.py

```diff
@@ -208,16 +207,9 @@
 IR_OPCODES: OpcodeMap = {**OPCODES, **PSEUDO_OPCODES}
 
 
-@contextlib.contextmanager
-def anchor_evm_version(evm_version: Optional[str] = None) -> Generator:
+def set_global_evm_version(evm_version: int) -> None:
     global active_evm_version
```

Comment:
Can this be combined into the global settings singleton?

Reply:
i think the change is a bit more extensive, but let me try